packages/client/src/pages/Game/Game.tsx

```diff
@@ -1,3 +1,104 @@
+import styled from 'styled-components';
+import { useMemo, useRef } from 'react';
+import { TSizes } from 'pages/Game/types/types';
+import { Player } from 'pages/Game/controllers/Player/Player';
+import { Ground } from 'pages/Game/controllers/Ground/Ground';
+import { useDidMount } from 'src/hooks/react';
+import { Platforms } from './controllers/Platforms/Platforms';
+
 export const Game = () => {
-  return <div>Game page</div>;
+  const sizes = useMemo<TSizes>(() => ({ width: 500, height: 600 }), []);
+  const canvasRef = useRef<HTMLCanvasElement | null>(null);
+  const canvasContextRef = useRef<CanvasRenderingContext2D | null>(null);
+  let player = new Player(canvasContextRef.current as CanvasRenderingContext2D, sizes);
+  let platforms = new Platforms(canvasContextRef.current as CanvasRenderingContext2D, sizes);
+  let ground = new Ground(canvasContextRef.current as CanvasRenderingContext2D, sizes);
+
+  const canvasInit = () => {
+    const canvas = canvasRef.current as HTMLCanvasElement;
+
+    canvasContextRef.current = canvas.getContext('2d');
```

Comment:
Я думаю, таких самонадеянных тайпкастов лучше избегать. 
Ты здесь перезаписываешь потенциальный null, как я понимаю. 
Лучше строчкой ниже проверяй, что у тебя нет null, например так:
```suggestion
    const canvas = canvasRef.current;

    canvasContextRef.current = canvas ? canvas.getContext('2d') : null;
```

Reply:
Исправил